tests/Fixer/Operator/BinaryOperatorSpacesFixerTest.php

```diff
@@ -447,6 +447,24 @@ public function provideTestCases()
                     'operators' => ['|' => BinaryOperatorSpacesFixer::NO_SPACE],
                 ],
             ],
+            [
+                '<?php
+$arr = array_map(function /* foo */ ($i): array {
```

Comment:
You added this test case as suggested by @kubawerlos but you removed the one you added previously. You should add both.